src/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -130,7 +131,12 @@ function thermal_sensors_widget_update(sender, data) {
                     .addClass('progress-bar-success');
             }
             // update bar
-            progressBar.html(sensor['temperature'] + ' &deg;C');
+            
+            if($('#thermal_sensors_widget_show_temp_in_fahrenheit').attr('checked') === 'checked') {  
+                progressBar.html((Number.parseFloat(((1.8) * sensor['temperature']+(32))).toFixed(1)) + ' &deg;F');
```

Comment:
Why not make this less complicated? `progressBar.html((Number.parseFloat(1.8 * sensor['temperature'] + 32).toFixed(1)) + ' &deg;F');`

Reply:
```suggestion
                progressBar.html(Number.parseFloat(1.8 * sensor['temperature'] + 32).toFixed(1) + ' &deg;F');
```